src/stores/configStore.js

```diff
@@ -63,6 +65,12 @@ export const initialConfigValues = {
   maxGroupCounts: 100,
   minLocalCounts: 50,
   minGlobalCounts: 100,
+  locationArray: ['region', 'country', 'division', 'location'],
```

Comment:
Remove – these won't ever change, and I've already defined these in a separate file, `src/constants/defs.json` as `GEO_LEVELS`